jekyll/_cci2/runner-faqs-adoc

```diff
@@ -0,0 +1,51 @@
+= CircleCI Runner FAQs
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== How do I get customers on to the preview?
+
+See How to run a CircleCI Runner preview
+
+=== Can I try runner out for myself?
+
+Internal users are welcome to give runner a try.
+
+The install docs can be found here:
+https://circleci-binary-releases.s3.amazonaws.com/circleci-launch-agent/docs/install.html
```

Comment:
Links with asciidoctor can be formatted as so:

`https://my-link.com[My link description].`

Reply:
Fixed